src/scss/objects/_lists.scss

```diff
@@ -51,12 +51,12 @@ ol {
 .c-list-ordered {
     > li {
         font-weight: $font-weight-bold;
-        @include font-size(mid);
+        @include font-size(heading-s);
```

Comment:
`heading-s` vs `body-l` - semantics again

Reply:
Have added a TODO to this to make sure this is addressed when we port the component (as it should have a different alias token at that point)